test/Bridges/Constraint/interval.jl

```diff
@@ -29,11 +29,9 @@ function test_split_basic()
         bridged_mock,
         config,
         include = [
-            "test_basic_$(F)_$(S)" for F in [
-                "VariableIndex",
-                "ScalarAffineFunction",
-                "ScalarQuadraticFunction",
-            ] for S in ["Interval", "EqualTo"]
+            "test_basic_$(F)_$(S)" for F in
```

Comment:
It's weird it's JuliaFormatter on my computer who changed that

Reply:
Different version maybe? It's 13.2 on the CI. I pinned it so we didn't have spurious changes. But we could revisit that.

Reply:
Yes, I have "JuliaFormatter v0.15.11", maybe we can update